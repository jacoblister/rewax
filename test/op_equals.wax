(func equals_test
  (print "equals_test:")
  (let compare str (alloc str))
  (<< compare "y")
  (<< compare "e")
  (<< compare "s")
  (let compare_result int (? (= "yes" compare) 1 0))
  (print compare_result)
)

(func main (result int)
  (call equals_test)
  (return 0)
)