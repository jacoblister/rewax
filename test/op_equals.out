equals_test:
1
